R/utils.R

```diff
@@ -20,3 +20,13 @@ quote_str <- function(x, quote = "\"") {
 
   paste0(quote, x, quote)
 }
+
+is_installed <- function(pkg) {
+  system.file(package = pkg) != ""
```

Comment:
Sure (this is originally from https://github.com/hadley/httr/blob/master/R/utils.r#L35)